src/aks-preview/HISTORY.md

```diff
@@ -5,6 +5,7 @@ Release History
 0.5.18
 +++++
 * Fix issue with node config not consuming logging settings
+* Remove azure-defender from list of available addons to install via `az aks enable-addons` command
```

Comment:
0.5.18 is existing version for aks-preview. Please upgrade to a new one, e.g. 0.5.19

Reply:
Done